xslt/Channels.xsl

```diff
@@ -37,49 +38,97 @@
 
 
     <xsl:template match="Intensity">
-        <xsl:if test="not(. = 'n/a')">
-            <xsl:element name="LightSourceSettings">
+        <xsl:variable name="inten" select="substring( ., 0, string-length(.)-1) div 100.0"/>
+        <xsl:if test="$inten &gt; 0.29">
```

Comment:
I know this is testing if intensity is greater than 0.29... which first do we have to use the the html char code? and second is that hyper specific to AICS data?

Reply:
1. It looks like using `>` works fine.
2. I don't know what the significance of 0.29.
@heeler, looks like you were the last person to touch these lines. Any input around Jackson's questions?

Reply:
I know that 0.29 comes up quite a lot in AICS data. Unsure where... I think it's the pixel sizes for some reason which is why I am confused. But also just hard coding a float value in a transform seems weird to me.

Reply:
Yeah so I'm having a hard time remembering exactly why 0.29 mattered. I'm guessing it was something Tim told me, or it was a way of filtering bad channels maybe? I'm pulling at straws. My suggestion is ask time if there's any significance to the value if not maybe I was testing something and forgot to remove it but I have a feeling I was told to do it.

Reply:
Thanks Jamie!

So I guess ask Tim if "0.29 intensity testing" is an AICS data thing or if it's a CZI thing... I truly think this was for testing purposes or something as I cant imagine hardcoding a value that isn't like 0, 1, or 100 in here.

Reply:
@heeler @JacksonMaxfield Tim isn't familiar with 0.29 intensity testing or anything like that, but he says 0.29 um is the value of the pipeline 100x objective z-step size. Not sure if that's just a coincidence.

I'm going to take a closer look at the output of this line and see if I can make sense of it.

Reply:
Hah. I knew it was related to the Z pixel size. I would fully remove that then. I can't imagine intensity is related to z pixel size and that sounds _very_ AICS specific and not general CZI.

Reply:
Errr.. Just remove the `if`?

Reply:
Yeah, I'll try to remove the `if`. The way `ID` is calculated here looks a little funky to me too, gonna take a look at that.